tests/MaintainersTest.php

```diff
@@ -209,28 +209,53 @@ public function testListMaintainers()
         $this->assertArrayHasKey('zendeskUrl', $maintainer);
         $this->assertNull($maintainer['zendeskUrl']);
     }
-    
-    public function testUserMaintainerUsers()
+
+    public function testLeastOneMemberLimit()
     {
-        $maintainers = $this->client->listMaintainers();
+        $maintainer = $this->client->createMaintainer([
+            'name' => self::TESTS_MAINTAINER_PREFIX . " - test least one member"
+        ]);
 
-        $this->assertGreaterThan(0, count($maintainers));
+        $maintainerId = $maintainer['id'];
 
-        $superAdmin = $this->client->verifyToken()['user'];
-        $maintainer = $maintainers[0];
+        $this->client->addUserToMaintainer($maintainerId, ['email' => $this->normalUser['email']]);
+
+        $members = $this->client->listMaintainerMembers($maintainerId);
+        $this->assertCount(2, $members);
+
+        $this->client->removeUserFromMaintainer($maintainerId, $this->superAdmin['id']);
+
+        $members = $this->client->listMaintainerMembers($maintainerId);
+        $this->assertCount(1, $members);
 
-        $maintainerMembers = $this->client->listMaintainerMembers($maintainer['id']);
+        try {
+            $this->client->removeUserFromMaintainer($maintainerId, $this->normalUser['id']);
+            $this->fail('The last member could not be removed from the maintainer');
+        } catch (ClientException $e) {
+            $this->assertEquals(400, $e->getCode());
+            $this->assertContains('least 1 member', $e->getMessage());
+        }
+
+        $members = $this->client->listMaintainerMembers($maintainerId);
+        $this->assertCount(1, $members);
+    }
+
+    public function testUserMaintainerUsers()
```

Comment:
UsersCanBeAddedAndRemoved...

A to nezmizelo? Já myslel, že to je jen JOIN/INVITE, ne?

Reply:
Nezmizelo, to se vyhodi az casem. Stejne jako prime pridavani useru do projektu a organizaci.